pkg/cloudprovider/cloud.go

```diff
@@ -56,9 +57,9 @@ type InformerUser interface {
 // Clusters is an abstract, pluggable interface for clusters of containers.
 type Clusters interface {
 	// ListClusters lists the names of the available clusters.
-	ListClusters() ([]string, error)
+	ListClusters(ctx context.Context) ([]string, error)
 	// Master gets back the address (either DNS name or IP address) of the master node for the cluster.
-	Master(clusterName string) (string, error)
+	Master(ctx context.Context, lusterName string) (string, error)
```

Comment:
The c in clusterName became a casualty of the refactor :)

Reply:
Good catch. Fixed.